test-scripts/ForgotPassword.rest

```diff
@@ -0,0 +1,17 @@
+POST http://localhost:8080/public/forgot-password
+Content-Type: application/json
+    
+{
+    "email":"ezgiozdikyar@gmail.com"
+}
+
+###
+
+POST http://localhost:8080/public/update-password
+Content-Type: application/json
+    
```

Comment:
Basic Auth kısmı eklenmeli.

Reply:
Bu şifre sıfırlama endpointi için. Kullanıcı bu kısıma geldiğinde şifresini bilmediği için authorization yok. 401 hatası dönmemesi için de public yapıldı.

Reply:
Anladımm tamamdır.